editor/components/post-author/check.js

```diff
@@ -17,7 +17,7 @@ import PostTypeSupportCheck from '../post-type-support-check';
 import { getCurrentPostType } from '../../store/selectors';
 
 export function PostAuthorCheck( { user, users, children } ) {
-	const authors = filter( users.data, ( { capabilities } ) => capabilities.level_1 );
+	const authors = filter( users.data, ( { capabilities } ) => get( capabilities, [ 'level_1' ], false ) );
```

Comment:
Nit: Lodash's `filter` performs a general truthiness check. `get` without a default value will return `undefined` if the path is not reachable. Since `undefined` is falsey, it would behave the same as an explicit `false`. In other words, the `, false` argument is redundant (though arguments to be made toward benefits of explicitness).